sql/src/main/scala/hydra/sql/JdbcRecordWriter.scala

```diff
@@ -214,6 +214,8 @@ class JdbcRecordWriter(val settings: JdbcWriterSettings,
     operations.clear()
   }
 
+  def rollback(): Unit = operations.clear()
```

Comment:
Looks good. More of a semantic change request: can we call this reset() (or something like that) instead of rollback since we are not technically rolling anything back?